src/OpenTelemetry.AutoInstrumentation.Native/netfx_assembly_redirection.h

```diff
@@ -14,145 +14,145 @@ void CorProfiler::InitNetFxAssemblyRedirectsMap()
-        { L"System.Net.WebSockets.Client", {4, 0, 2, 0} },
-        { L"System.Numerics.Vectors", {4, 1, 4, 0} },
-        { L"System.ObjectModel", {4, 0, 11, 0} },
-        { L"System.Reflection", {4, 1, 2, 0} },
-        { L"System.Reflection.Extensions", {4, 0, 1, 0} },
-        { L"System.Reflection.Primitives", {4, 0, 1, 0} },
-        { L"System.Resources.Reader", {4, 0, 2, 0} },
-        { L"System.Resources.ResourceManager", {4, 0, 1, 0} },
-        { L"System.Resources.Writer", {4, 0, 2, 0} },
-        { L"System.Runtime", {4, 1, 2, 0} },
-        { L"System.Runtime.CompilerServices.Unsafe", {6, 0, 0, 0} },
-        { L"System.Runtime.CompilerServices.VisualC", {4, 0, 2, 0} },
-        { L"System.Runtime.Extensions", {4, 1, 2, 0} },
-        { L"System.Runtime.Handles", {4, 0, 1, 0} },
-        { L"System.Runtime.InteropServices", {4, 1, 2, 0} },
-        { L"System.Runtime.InteropServices.RuntimeInformation", {4, 0, 2, 0} },
-        { L"System.Runtime.Numerics", {4, 0, 1, 0} },
-        { L"System.Runtime.Serialization.Formatters", {4, 0, 2, 0} },
-        { L"System.Runtime.Serialization.Json", {4, 0, 1, 0} },
-        { L"System.Runtime.Serialization.Primitives", {4, 2, 0, 0} },
-        { L"System.Runtime.Serialization.Xml", {4, 1, 3, 0} },
-        { L"System.Security.Claims", {4, 0, 3, 0} },
-        { L"System.Security.Cryptography.Algorithms", {4, 3, 0, 0} },
-        { L"System.Security.Cryptography.Csp", {4, 0, 2, 0} },
-        { L"System.Security.Cryptography.Encoding", {4, 0, 2, 0} },
-        { L"System.Security.Cryptography.Primitives", {4, 0, 2, 0} },
-        { L"System.Security.Cryptography.X509Certificates", {4, 1, 2, 0} },
-        { L"System.Security.Principal", {4, 0, 1, 0} },
-        { L"System.Security.SecureString", {4, 1, 0, 0} },
-        { L"System.Text.Encoding", {4, 0, 11, 0} },
-        { L"System.Text.Encoding.Extensions", {4, 0, 11, 0} },
-        { L"System.Text.Encodings.Web", {7, 0, 0, 0} },
-        { L"System.Text.Json", {7, 0, 0, 2} },
-        { L"System.Text.RegularExpressions", {4, 1, 1, 0} },
-        { L"System.Threading", {4, 0, 11, 0} },
-        { L"System.Threading.Overlapped", {4, 1, 0, 0} },
-        { L"System.Threading.Tasks", {4, 0, 11, 0} },
-        { L"System.Threading.Tasks.Extensions", {4, 2, 0, 1} },
-        { L"System.Threading.Tasks.Parallel", {4, 0, 1, 0} },
-        { L"System.Threading.Thread", {4, 0, 2, 0} },
-        { L"System.Threading.ThreadPool", {4, 0, 12, 0} },
-        { L"System.Threading.Timer", {4, 0, 1, 0} },
-        { L"System.ValueTuple", {4, 0, 3, 0} },
-        { L"System.Xml.ReaderWriter", {4, 1, 1, 0} },
-        { L"System.Xml.XDocument", {4, 0, 11, 0} },
-        { L"System.Xml.XmlDocument", {4, 0, 3, 0} },
-        { L"System.Xml.XmlSerializer", {4, 0, 11, 0} },
-        { L"System.Xml.XPath", {4, 0, 3, 0} },
-        { L"System.Xml.XPath.XDocument", {4, 1, 0, 0} },
+		{ L"Grpc.Core", {2, 0, 0, 0} },
```

Comment:
If someone can test it locally, currently indentation seems more like Github error 🤔